src/cockpit/packages.py

```diff
@@ -227,22 +259,31 @@ def get_content_security_policy(self, origin):
     def serve_file(self, path, channel):
         filename = self.negotiate_file(path, channel.headers)
 
-        if filename is None:
+        if filename:
+            with (self.path / filename).open('rb') as file:
+                data = file.read()
+        # HACK: if a translation file is missing, just return empty
```

Comment:
I think it reads better if this is inside the elif block, but it's a matter of taste.